tests/test.py

```diff
@@ -58,18 +58,38 @@ def setUpClass(cls):
             cls.ctx.config['macro_paths'][0],
             f'{cls.macro_name}.sql',
         )
+        cls.dbt_seed = (
+            'dbt seed'
+            f' --project-dir {cls.project_dir}'
+            f' --profiles-dir {cls.project_dir}'
+            f' --target-path {cls.project_dir}/target'
```

Comment:
I added the `--target-path` option to all `dbt` commands than update the `manifest.json` file so that I could run dbt from various directories. By default dbt will write targets to the top level directory that `dbt` is being executed from (see https://docs.getdbt.com/reference/project-configs/target-path)

Reply:
Makes sense. So then do we need to add the `--target-path` option to the definitions of `cls.dbt_snapshot` and `cls.dbt_compile` on the lines below? I noticed the option was present for `cls.dbt_seed` and `cls.dbt_run`, but not the others. (Sorry in advance if I am forgetting something about the snapshot and compile commands that makes doing this unnecessary)